timemachine/cpp/src/context.cu

```diff
@@ -151,6 +152,8 @@ void Context::_step(double lambda, unsigned long long *du_dl_out) {
     // the observables decide on whether or not to act on given
     // data (cheap pointers in any case)
 
+    cudaStream_t stream = static_cast<cudaStream_t>(0);
```

Comment:
Seemed cleaner to reuse the stream through out a step, to enable async all the way through